tritium-processor/src/test/java/com/palantir/tritium/examples/DelegateToRunnable.java

```diff
@@ -17,6 +17,8 @@
 package com.palantir.tritium.examples;
 
 import com.palantir.tritium.annotations.Instrument;
+import com.palantir.tritium.annotations.Proxy;
 
+@Proxy
 @Instrument
 public interface DelegateToRunnable extends Runnable {}
```

Comment:
Can you add a test for multiple interfaces being extended? *I think* you have a lot of logic to resolve any overrides etc. might be useful to put that through it's paces.

Reply:
:+1: